src/node/internal/zlib.d.ts

```diff
@@ -250,3 +248,13 @@ export class BrotliEncoder extends CompressionStream {
   ): boolean;
   public params(): void;
 }
+
+export const enum ZlibMode {
```

Comment:
Enum values are not 1 to 1 representably Node.js. We should avoid using them. Can we convert this to an object and move this to the file where it is used?

Reply:
I moved this enum to internal_zlib.ts, but I was not sure how to replace it with an object, because `keyof typeof CLASS_BY_MODE` is just `number`.